### PR TITLE
FR - update stationery and variety_store brands

### PR DESCRIPTION
**Pull request by nlehuby**

### CODE REVIEW

File: rules.csv [hunk 1]
```diff
@@ -916,9 +916,9 @@ FR,supermarket,Magasins U,Q2529029,open_adapted,,,,https://www.magasins-u.com/ac
 FR,supermarket,Super U,Q2529029,open_adapted,,,,https://www.magasins-u.com/accueil,https://www.magasins-u.com/fichiers/horaires-magasins_pdf
 FR,supermarket,Système U,Q2529029,open_adapted,,,,https://www.magasins-u.com/accueil,https://www.magasins-u.com/fichiers/horaires-magasins_pdf
 FR,supermarket,U Express,Q2529029,open_adapted,,,,https://www.magasins-u.com/accueil,https://www.magasins-u.com/fichiers/horaires-magasins_pdf
-FR,variety_store,Action,Q2634111,open,,,,https://www.action.com/fr-fr/mise-a-jour/,
-FR,variety_store,GiFi,Q3105439,open,,,,https://www.gifi.fr/,
-FR,variety_store,Noz,Q3345688,open,,,,https://www.nozarrivages.com/,https://www.nozarrivages.com/magasin/
+FR,variety_store,Action,Q2634111,open,,,Uniquement pour les produits essentiels,https://www.action.com/fr-fr/mise-a-jour/,https://www.action.com/fr-fr/points-de-vente/
+FR,variety_store,GiFi,Q3105439,partial,,,Click-and-collect sur https://www.gifi-clickandcollect.fr/,https://www.gifi.fr/faq-covid19.html,
```
Comment (nlehuby) [inline rules.csv:920]: Do we already have a consensus about the tag to use for the click&collect url ?

Comment (vinber) [reply rules.csv:920]: I don't think so

Comment (overflorian) [reply rules.csv:920]: Nope. I started a discussion on talk-fr mailing-list and created a dedicated issue that needs to be discussed and implemented very soon https://github.com/caresteouvert/caresteouvert/issues/841

Comment (nlehuby) [reply rules.csv:920]: Ok, so let's add it in the description field for now, and we will move it to an appropriate tag later.

**Pull request merged**